enterprise/frontend/src/metabase-enterprise/caching/components/CacheSection/CacheSection.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useCallback, useState } from "react";
+import { t } from "ttag";
+
+import TippyPopoverWithTrigger from "metabase/components/PopoverWithTrigger/TippyPopoverWithTrigger";
+import Button from "metabase/core/components/Button";
+import ActionButton from "metabase/components/ActionButton";
+import NumericInput from "metabase/core/components/NumericInput";
+import { color } from "metabase/lib/colors";
+
+import { normalizeCacheTTL } from "../../utils";
+
+import { Text, CacheSectionRoot, CachePopover } from "./CacheSection.styled";
+
+type CACHE_TTL = number | null;
```

Comment:
We don't use caps for types. It also doesn't really introduce a new type. Could this be inlined instead? You also don't need to set the type in `useState`